with-firebase-storage-upload/App.js

```diff
@@ -0,0 +1,203 @@
+import * as ImagePicker from 'expo-image-picker';
+import * as Permissions from 'expo-permissions';
+import * as firebase from 'firebase';
+import React from 'react';
+import {
+  ActivityIndicator,
+  Button,
+  Clipboard,
+  Image,
+  Share,
+  StatusBar,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import uuid from 'uuid';
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyAlZruO2T_JNOWn4ysfX6AryR6Dzm_VVaA',
+  authDomain: 'blobtest-36ff6.firebaseapp.com',
+  databaseURL: 'https://blobtest-36ff6.firebaseio.com',
+  storageBucket: 'blobtest-36ff6.appspot.com',
+  messagingSenderId: '506017999540',
+};
+
+firebase.initializeApp(firebaseConfig);
```

Comment:
Support fast refresh
```suggestion
if (!firebase.apps.length) {
  firebase.initializeApp(firebaseConfig);
}
```

Reply:
Awesome!